man/fi_domain.3.md

```diff
@@ -446,8 +446,8 @@ transfer operation.
   seen by the initiator of a request.  For FI_EP_DGRAM endpoints, if the target EP
   queues are unable to accept incoming messages, received messages will
   be dropped.  For reliable endpoints, if RM is disabled, the transmit
-  operation will complete in error.  If RM is enabled, the provider will
-  internally retry the operation.
+  operation will complete with the error code FI_ENORX.  If RM is enabled, the
```

Comment:
Is every RM disabled provider able to tell that a transmit error is caused by no matching recv buffer? Forcing the error code here may be over restrictive.

Reply:
I'm not sure, I suppose that could be the case depending on the device. Perhaps something like this would be better?

"For reliable endpoints, if RM is disabled, the transmit operation will complete in error. A provider may choose to return an error completion with the error code FI_ENORX for that transmit operation so that it can be retried."

Reply:
Yes, that looks better.

Reply:
Thanks, updated.